### DISCUSSION TITLE
Active Directory User Login

### DISCUSSION THREAD

**Discussion by drakub** [YetiForceCompany/YetiForceCRM#15323] [General]
After configuring the LDAP settings, do I need to create the users in the CRM first or the creating the users in the CRM is not necessary?

**Comment by mariuszkrzaczkowski** [comment]
the user must exist in CRM and have the same username as in LDAP

**Reply by drakub** [reply]
But this is not working form me. I am on YetiForce 6.2 and the environment is Linux distro (Ubuntu)

The same configurations work perfect on 6.1 installed earlier

**Reply by mariuszkrzaczkowski** [reply]
check logs

**Comment by drakub** [comment]
Which logs particularly, server error logs or YetiForce and if YetiForce,
where are these logs kept

**Reply by mariuszkrzaczkowski** [reply]
https://yetiforce.com/pl/component/zoo/item/debugging
